core/org.openjdk.jmc.rjmx.common/src/main/java/org/openjdk/jmc/rjmx/common/services/jfr/package-info.java

```diff
@@ -64,9 +64,9 @@
  * 	IFlightRecorderService jfr = handle.getServiceOrThrow(IFlightRecorderService.class);
  *
  * 	long duration = 5000;
- * 	IConstrainedMap<String> defaultRecordingOptions = jfr.getDefaultRecordingOptions();
- *  IDescribedMap<EventOptionID> defaultEventOptions = jfr.getDefaultEventOptions();
- * 	IConstrainedMap<String> recordingOptions = new RecordingOptionsBuilder(defaultRecordingOptions.mutableCopy())
+ * 	IConstrainedMap&lt;String&gt; defaultRecordingOptions = jfr.getDefaultRecordingOptions();
+ *  IDescribedMap&lt;EventOptionID&gt; defaultEventOptions = jfr.getDefaultEventOptions();
```

Comment:
Maybe you can fix the indentation in this line, too.

Reply:
Ah thanks again for the keen eye